### DISCUSSION TITLE
How to set SpaceBetween SpaceEvenly in Vbox And HBox

### DISCUSSION THREAD

**Discussion by bobwatcherx** [widgetti/solara#99] [Q&A]
how to use Spacebetween spaceEvenly 
give me example code

**Comment by maartenbreddels** [comment]
Does https://solara.dev/api/row help? Some older examples use vbox and hbox, I prefer the clearer name column and row

**Reply by bobwatcherx** [reply]
if there is already column and row . then why are there vbox and hbox . try to explain between vbox and column

**Reply by maartenbreddels** [reply]
VBox and HBox are the old names, I think we should deprecate them
